isis/src/base/apps/topds4/topds4.cpp

```diff
@@ -197,3 +229,34 @@ namespace Isis {
     }
   }
 }
+
+
+QString PDS4PixelType(Isis::PixelType ipixelType, Isis::ByteOrder ibyteOrder) {
```

Comment:
This is xml output dependent, so it will have to change when we adapt to other output types.
Why not put this in the template? Does the template logic not have the ability to do this?

Reply:
This could be included in the template, but I think it's better suited as a callback because any template for exporting to xml will need to use this same chunk of logic.

Reply:
let's go ahead and do it this way for now. We can change later if needed.